hexrd/material.py

```diff
@@ -684,9 +684,9 @@ def vol_per_atom(self):
     def atom_pos(self):
         return self.unitcell.atom_pos
 
-    @property
-    def atom_type(self):
-        return self.unitcell.atom_type
+    # @property
+    # def atom_type(self):
+    #     return self.unitcell.atom_type
```

Comment:
@saransh13 -- does removing this property break anything?

Reply:
Ran fine for me. I noticed it was doubly defined. There is an `atomtypes`
property on `material.Material` which is actually doing the updating.